### PR TITLE
add regular progress-bar

### LINKED ISSUE #7: Add a progress bar to the dataset upload on the /contribute page
You may need to fork this snippet to get a snippet that supports up to 500 samples

Note that the 499 in the screenshot is wrong and should also be fixed, it's off by one

Image

/bounty $10

### PR DESCRIPTION
**Pull request by Anshgrover23**
/claim #7 
Fixes #7 

### Screenshots:
image

### CODE REVIEW

**Review by seveibar — CHANGES_REQUESTED** [review]
use a regular progress bar

**Review by seveibar — CHANGES_REQUESTED** [review]
implement using tailwind, we don't need a separate library for this

**Comment by Anshgrover23** [comment]
@seveibar ready now using tailwind

File: components/contribute/ContributeProcessingStep.tsx [hunk 1]
```diff
@@ -98,13 +97,19 @@ export function ContributeProcessingStep({
...
+            style={{ width: `${samplesProcessed}%` }}
+          ></div>
+        </div>
         <div className="text-sm text-muted-foreground">
           Processing sample{" "}
           {Math.floor(
             (sampleRange.end - sampleRange.start) * (samplesProcessed / 100),
           )}{" "}
-          of {sampleRange.end - sampleRange.start}
+          of {sampleRange.end}
```
Comment (seveibar) [inline components/contribute/ContributeProcessingStep.tsx:112]:
```suggestion
          of {sampleRange.end - sampleRange.start + 1}
```

**Review by seveibar — APPROVED** [review]

**Pull request merged**